bazel/utils/template.bzl

```diff
@@ -88,3 +88,42 @@ template_tool = attr.label(
     executable = True,
     allow_single_file = True,
 )
+
+def expand_template_binary_impl(ctx):
```

Comment:
Normally, common util functions, especially for bazel, should be defined in the `enkit` repo so that these util functions can be used by the `enkit` repo and imported by the `internal` repo. I will probably replace the template function defined in the `internal` repo with the one I defined in the `enkit` repo in the future.